core/bug_api.php

```diff
@@ -84,6 +84,16 @@ class BugData {
 	protected $id;
 
 	/**
+	 * Alias
+	 */
+	protected $alias = null;
```

Comment:
Why?

Almost every other implementation allow to specify it from client side. Calculation from server side will be too much complicated. 

For example, should it depend on version? E.g. does application wants to collect bugs for each its version individually? Does application want to consider the bug with same alias in a version after bug was fixed to be a new bug or the same bug? 

When alias is defined by application this can be easily customizable by including or excluding version field from calculating bug alias/hash.

However, if you want to make server calculation - then you have to add many options to include/exclude the following pieces:
- Exception address
- Exception type
- Exception message
- Exception module
- Project module
- Version for both of the above
- Call stack - either full or partial

Mantis simply have no access to this information, whereas clien application - does.